tests/rbd_mirror/test_rbd_mirror_snapshot.py

```diff
@@ -41,10 +41,15 @@ def run(**kw):
         snapshot_schedule_level = config.get("snapshot_schedule_level")
         if not snapshot_schedule_level or snapshot_schedule_level == "cluster":
             mirror1.mirror_snapshot_schedule_add()
+            mirror1.mirror_snapshot_schedule_add(interval="24h")
         elif snapshot_schedule_level == "pool":
             mirror1.mirror_snapshot_schedule_add(poolname=poolname)
+            mirror1.mirror_snapshot_schedule_add(poolname=poolname, interval="24h")
         else:
             mirror1.mirror_snapshot_schedule_add(poolname=poolname, imagename=imagename)
+            mirror1.mirror_snapshot_schedule_add(
+                poolname=poolname, imagename=imagename, interval="24h"
```

Comment:
Can we have a shorter interval say 5mins or so, so that if we verify snapshots created after the interval (which in my opinion we should as part of verify snapshot schedule, we will be able to see a snapshot without having to wait for 24h since such a long duration is not acceptable in cephci)

Reply:
Also have interval given as an input as well, and add a default value only when input is not given?